pages/instructorQuestionSettings/instructorQuestionSettings.js

```diff
@@ -169,6 +230,74 @@ router.get('/', function(req, res, next) {
                 callback(null);
             });
         },
+        (callback) => {
+            const clientFilesCoursePath = encodePath(path.join(res.locals.course.path, 'clientFilesCourse', 'thumbnails'));
+            const questionPath = encodePath(path.join(res.locals.course.path, 'questions', res.locals.question.qid));
+            const publicPath = encodePath('public/images/thumbnails');
+            res.locals.available_thumbnails = [];
+            async.series([
+                (callback) => {
+                    fs.readdir(clientFilesCoursePath, function (err, files) {
+                        // Probably no clientFilesCourse directory
+                        if (err) {
```

Comment:
When I tested this change, the error triggered on ENOENT and caused the server to exit, so I added a special case for it. I want ENOENT to call callback(null, {}) because a clientFilesCourse/thumbnails not existing is completely OK and shouldn't cause an error.

Reply:
Ah gotcha, disregard my comments then.